include/functions.h

```diff
@@ -1120,22 +1120,22 @@ void BgCheck2_UpdateActorPosition(CollisionContext* colCtx, s32 index, Actor* ac
 void BgCheck2_UpdateActorYRotation(CollisionContext* colCtx, s32 index, Actor* actor);
 void BgCheck2_AttachToMesh(CollisionContext* colCtx, Actor* actor, s32 index);
 u32 BgCheck2_UpdateActorAttachedToMesh(CollisionContext* colCtx, s32 index, Actor* actor);
-void BcCheck3_BgActorInit(DynaPolyActor* actor, UNK_TYPE4 param_2);
-void BgCheck3_LoadMesh(GlobalContext* globalCtx, DynaPolyActor* actor, CollisionHeader* meshHeader);
-void BgCheck3_ResetFlags(DynaPolyActor* actor);
-void func_800CAE88(DynaPolyActor* actor);
-void func_800CAE9C(DynaPolyActor* actor);
-void func_800CAEB0(CollisionContext* colCtx, s32 index);
-void func_800CAEE0(DynaPolyActor* actor);
-void func_800CAEF4(CollisionContext* colCtx, s32 index);
-void func_800CAF24(DynaPolyActor* actor);
-void func_800CAF38(DynaPolyActor* actor);
-s32 func_800CAF4C(DynaPolyActor* actor);
-s32 func_800CAF70(DynaPolyActor* actor);
-s32 func_800CAF94(DynaPolyActor* actor);
-s32 func_800CAFB8(DynaPolyActor* actor);
-s32 func_800CAFDC(DynaPolyActor* actor);
-// void func_800CB000(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE2 param_5);
+void DynaPoly_Init(DynaPolyActor* dynaActor, s32 flags);
```

Comment:
I'm unsure about simply `DynaPoly` for these functions. In oot, we use `DynaPolyActor` for these as they are external functions for the `DynaPolyActor` structure, and we use `DynaPoly` in `z_bgcheck.c` for functions that manage the dynapoly system as a whole.

Reply:
These manage the actor, so I'd have to no issues with that. It is a bit too long, though. Wonder if it'd work to merge the two systems.